app/components/works/contributor_row_component.rb

```diff
@@ -62,6 +70,14 @@ def orcid?
       model.orcid.present?
     end
 
+    def person?
+      model.contributor_type == "person" || model.contributor_type.blank?
+    end
+
+    def organization?
+      model.contributor_type == "organization"
+    end
+
```

Comment:
let's us figure out if the model object represents a person or organization (default to person for a new row)